docs/source/reference/kubernetes/examples/kueue-example.rst

```diff
@@ -237,32 +202,35 @@ Here, a cluster queue and a local queue are created.
     spec:
       clusterQueue: "skypilot-cluster-queue"
 
-The team submit their jobs to the local queue ``skypilot-local-queue``.
-The jobs from the team are subject to the same quota defined in the cluster queue,
-and the jobs from the team are ordered by their priority together.
+To create the cluster and local queues above, save the snippet to ``kueue.yaml`` and run the following command:
+
+.. code-block:: bash
+
+    # if 'skypilot' namespace does not exist, create it
+    kubectl create namespace skypilot
+    # create the cluster and local queue
+    kubectl apply -f kueue.yaml
+
+
+The team submits jobs to the local queue ``skypilot-local-queue``.
+The jobs from the team are subject to the quota defined in the cluster queue,
+and the jobs from the team are ordered by their priority.
```

Comment:
I know I already commented on this but it still feels weird.
We can just omit "team" altogether.
```suggestion
Now, jobs can be submitted to the local queue ``skypilot-local-queue``.
Jobs are subject to the quota defined in the cluster queue,
and the jobs are ordered by their priority.
```